### ISSUE TITLE
System.Web.Http.dll should not be distributed with the NuGet package

### ISSUE THREAD

**Issue by JohannesRudolph** [dbettin/webapi.nlog#1]
... as it conflicts with the released version of WebApi (and is legally questionable due to licensing issues...). Can you instead make the nuget package depend on Microsoft.AspNet.WebApi.Core.4.0 and above?

**Comment by jonswaino** [comment]
I agree, I've included this in my project and now getting lots of errors like the one below:

Error   13  'System.Web.Http.HttpConfiguration' does not contain a definition for 'Services' and no extension method 'Services' accepting a first argument of type 'System.Web.Http.HttpConfiguration' could be found (are you missing a using directive or an assembly reference?)

**Comment by Allen-Wei** [comment]
Do you have resolved the Error 13?

**Comment by alanwei43** [comment]
Have you resolved the Error 13?
